uebung3.py

```diff
@@ -0,0 +1,33 @@
+import math
+
+N = 3
+middle = (N*3)+2
+
+## Row 1
+print(" " * (middle-1), end="")
+print("y", end="")
+print(" " * (middle-1),end="\n\n")

```

Comment:
you're using the expression `" " * (middle-1)` a lot, I'd recommend you store it in a constant for performance, brevity and readability. Then you can also do: `print(" " * (middle-1) + "y" + " " * (middle-1) + "\n")` without having to work with the `end`